src/App.jsx

```diff
@@ -2,15 +2,27 @@ import "./App.css";
 
 import SideBar from "./components/SideBar/SideBar";
 import TabBar from "./components/TabBar/TabBar";
-import Experience from "./components/Experience";
-
+import AnimationsBadge from "./components/AnimationsBadge/AnimationsBadge";
+import AnimationsStatus from "./components/AnimationsStatus/AnimationsStatus";
 
 function App() {
   return (
     <>
-      <SideBar />
-      <TabBar />
-      <Experience />
+      <div className="relative h-screen ">
+        <SideBar />
+        <TabBar />
+
+        <div className="absolute bottom-8  w-full flex items-center">
+          <div className="ml-8">
+            <AnimationsBadge />
+          </div>
+          <div className=" absolute left-1/2 transform -translate-x-1/2 ">
+            <AnimationsStatus />
+          </div>
```

Comment:
I think it would be better if the status disappears after 2 seconds. Also if enabled again "Animation started" would be helpful.

Reply:
thanks, is changed